lib/parallel_tests/cli.rb

```diff
@@ -75,6 +75,7 @@ def run_tests_in_parallel(num_processes, options)
           end
         end
 
+        puts options.inspect
```

Comment:
```suggestion
```

Reply:
Whoops, don't want that either. Removed in 3a62cb0